<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiLinkShipWidget</class>
 <widget class="QWidget" name="uiLinkShipWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uiLinkShipWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="4,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_ShipID"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Link">
         <property name="text">
          <string>Link</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5">
       <item>
        <widget class="QLabel" name="label_ErrorTitle">
         <property name="text">
          <string>Error:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Error">
         <property name="text">
          <string>Null</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
